{
	"id": "AVG-1691",
	"advisories": [
		{
			"content": {
				"id": "AVG-1691",
				"severity": [
					{
						"type": "vendor",
						"source": "security.archlinux.org",
						"vendor": "High"
					}
				],
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/AVG-1691"
					}
				]
			},
			"ecosystems": [
				"arch"
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2021-28363",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-28363"
					}
				]
			},
			"ecosystems": [
				"arch"
			]
		}
	],
	"detection": [
		{
			"ecosystem": "arch",
			"criteria": {
				"operator": "OR",
				"criterions": [
					{
						"vulnerable": true,
						"fix_status": {
							"class": "fixed"
						},
						"package": {
							"name": "python-urllib3"
						},
						"affected": {
							"type": "pacman",
							"range": [
								{
									"lt": "1.26.4-1"
								}
							],
							"fixed": [
								"1.26.4-1"
							]
						}
					}
				]
			}
		}
	],
	"data_source": {
		"id": "arch",
		"raws": [
			"vuls-data-raw-arch/AVG-1691.json"
		]
	}
}
